setup.cfg

```diff
@@ -32,21 +32,20 @@ install_requires =
     more-itertools~=8.0
     nglview~=3.0
     optimade-client==2022.9.19
-    scikit-learn~=0.24
     spglib~=1.14
     vapory~=0.1.2
 python_requires = >=3.8
 include_package_data = True
 zip_safe = False
 
 [options.extras_require]
+smiles =
```

Comment:
I am not sure about this, if you want "smiles" related packages installed by `aiidalab install`, it not supports install from extras dependencies yet.

Reply:
That's okay I think, because I can declare the dependency in my setup.cfg with these extras dependencies, so it should pull the package from pip.

Reply:
Ah, this is clever, but only for packages like `aiidalab-widgets-base` and when we not treat it as "app" anymore. Anyway, good for the change here.

Reply:
Yes, that is a good point. But @yakutovicha said many times he wants to turn AWB into library anyways, so it should be okay.